app/src/main/java/com/sklein/ubimo_pager/MainActivity.java

```diff
@@ -1,32 +1,44 @@
 package com.sklein.ubimo_pager;
 
+import android.content.Context;
 import android.content.Intent;
 import android.media.AudioAttributes;
+import android.media.AudioManager;
 import android.media.MediaPlayer;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
 import android.view.View;
+import android.view.WindowManager;
 import android.widget.ImageButton;
 import android.widget.TextView;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        getWindow().addFlags(WindowManager.LayoutParams.FLAG_SHOW_WHEN_LOCKED);
         super.onCreate(savedInstanceState);
         Intent smsIntent = getIntent();
         Bundle smsBundle = smsIntent.getExtras();
         if (null != smsBundle) {
             setContentView(R.layout.activity_main);
 
+            final AudioManager audio = (AudioManager) getSystemService(Context.AUDIO_SERVICE);
+            final int music = AudioManager.STREAM_MUSIC;
```

Comment:
* You can use `AudioManager.STREM_MUSIC` enum directly in `setStreamVolume()` call, it's a little more verbose but also more readable.
* Save the current volume value with `getStreamVolume()` before setting it on maximum
* Revert the volume to original on `onClickListener()`